src/components/templates/ArtPiece/ArtPiece.tsx

```diff
@@ -18,62 +20,83 @@ import SparkleFairiesPopUp from "components/molecules/SparkleFairiesPopUp/Sparkl
 
 import "./ArtPiece.scss";
 
+const DECLARATIVE_ASPECT_RATIOS = [
+  `${VideoAspectRatio.sixteenNine}`,
+  `${VideoAspectRatio.anamorphic}`,
+  `${VideoAspectRatio.fullWidth}`,
```

Comment:
No need for `${}`

Reply:
Yes there is. TS complains enums aren't strings, so a bit of forcing our or its hand...